.github/workflows/build.yml

```diff
@@ -47,6 +47,7 @@
             docker-compose up -d && sleep 5s
         - name: Check Docker Status
           run: |
+            sleep 20s
             status=$(docker ps --all --filter name=orthanc-server --format '{{.Status}}')
```

Comment:
maybe you should check the status of the postgres container as well

Reply:
Definitely. I feel like this has to do with Postgres not fully initialized before Orthanc does as this sometimes happens on my own machine as well when doing a first-time launch of docker-compose.